### PR TITLE
Add mintable function

### PR DESCRIPTION
**Pull request by tstxni97**
Update the client to have mintable functions.

### CODE REVIEW

File: client/erc721/mainnet_erc721_contract.go [hunk 1]
```diff
@@ -48,6 +48,18 @@ func (c *MainnetERC721Client) OwnerOf(tokenID *big.Int) (common.Address, error)
 	return c.contract.OwnerOf(nil, tokenID)
 }
 
+func (c *MainnetERC721Client) Exists(tokenID *big.Int) (bool, error) {
+	return c.contract.Exists(nil, tokenID)
+}
+
+func (c *MainnetERC721Client) Minto(caller *client.Identity, to common.Address, amount *big.Int) error {
```
Comment (enlight) [inline client/erc721/mainnet_erc721_contract.go]: `Minto` -> `MintTo`

File: client/erc20/mainnet_erc20_client.go [hunk 2]
```diff
@@ -52,6 +52,14 @@ func (c *MainnetERC20Client) Transfer(caller *client.Identity, to common.Address
 	return client.WaitForTxConfirmation(context.TODO(), c.ethClient, tx, c.TxTimeout)
 }
 
+func (c *MainnetERC20Client) Minto(caller *client.Identity, to common.Address, amount *big.Int) error {
```
Comment (enlight) [inline client/erc20/mainnet_erc20_client.go]: `Minto` -> `MintTo`

File: client/erc721/mainnet_erc721_contract.go (hunk 1 above)
Comment (phoorichet) [reply client/erc721/mainnet_erc721_contract.go]: Fixed

File: client/erc20/mainnet_erc20_client.go (hunk 2 above)
Comment (phoorichet) [reply client/erc20/mainnet_erc20_client.go]: Fixed

File: builtin/types/transfer_gateway/transfer_gateway.proto [hunk 3]
```diff
@@ -535,4 +534,4 @@ message TransferGatewayResubmitWithdrawalError {
 
 message TransferGatewayUpdateBinanceTransferFeeRequest{
     BigUInt transfer_fee = 1;
-}
\ No newline at end of file
+}
```
Comment (enlight) [inline builtin/types/transfer_gateway/transfer_gateway.proto]: Seeing as the protobufs didn't need changing after all revert any changes to this file with `git checkout master -- builtin/types/transfer_gateway.proto` & same with the `.pb.go`.

File: client/erc721/mainnet_erc721_contract.go [hunk 4]
```diff
@@ -48,6 +48,18 @@ func (c *MainnetERC721Client) OwnerOf(tokenID *big.Int) (common.Address, error)
 	return c.contract.OwnerOf(nil, tokenID)
 }
 
+func (c *MainnetERC721Client) Exists(tokenID *big.Int) (bool, error) {
+	return c.contract.Exists(nil, tokenID)
+}
```
Comment (enlight) [inline client/erc721/mainnet_erc721_contract.go]: You removed this public `exists` function in the Solidity contract, didn't you?

Comment (tstxni97) [reply client/erc721/mainnet_erc721_contract.go]: yea I removed the function. This should be removed too

**Pull request merged**